src/components/common/Layout/Layout.jsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
 import Header from '../Header/Header';
+import useAuth from 'hooks/useAuth';
+import Map from 'components/common/Map/Map';
+import PhoneAppSection from 'components/common/PhoneAppSection/PhoneAppSection';
 
 const Layout = ({ children }) => {
+  const { authenticated } = useAuth();
+
   return (
     <>
       <Header />
       {children}
+      {authenticated ? (
```

Comment:
This check seems unnecessary as logging out will redirect you to the Login page by default.

Reply:
But it's not. This conditional will return the iphone section if you are not logged in and a map if you are. 
It's a Layout not to repeat this sections in every page. 

Do you know a better way to do this?

Reply:
<img width="516" alt="image" src="https://user-images.githubusercontent.com/38699660/164054941-99ebd740-d309-4d74-9991-339ceec6e326.png">
Well the one I implemented was this, although I do have the issue of the map re-rendering when something changes on the left side. The logout part works all the same though.